.github/Jeda-1101-iter_rec.py

```diff
@@ -0,0 +1,13 @@
+"""
+base:int or float
+exp: int >= 0
+returns: int or float, base^exp
+"""
+base = 0
+exp = 0
+def iterativePower(base, exp):
+    if exp == 1:
+        return base
+    else:
+        return base * (iterativePower(base, exp-1))
+print(iterativePower(3,5))
```

Comment:
You need to write two functions:

iterativePower using LOOPS
recursivePower using RECURSION @jedapanita